src/operator/tensor/elemwise_binary_op_basic.cu

```diff
@@ -36,21 +36,21 @@ NNVM_REGISTER_OP(_backward_add)
   ElemwiseBinaryOp::BackwardUseNoneWithHalf2<gpu, mshadow_op::identity,
     mshadow_op::identity>);
 
-NNVM_REGISTER_OP(_sub)
+NNVM_REGISTER_OP(elemwise_sub)
```

Comment:
Doesn't this break backward compatibility in cpp package?

Reply:
not that I know of